Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -265,6 +265,35 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
 
     // TODO: Display proper error messaging from the backend
     self.didBlockUserError = blockUserEvent.errors().ignoreValues()
+
+    // MARK: User Blocking Analytics
+
+    _ = self.blockUserProperty.signal
+      .combineLatest(with: project)
+      .observeValues { blockedUserId, project in
+        AppEnvironment.current.ksrAnalytics
+          .trackBlockedUser(
+            project,
+            page: .project,
+            sectionContext: .comments,
+            typeContext: .initiate,
+            targetUserId: "\(blockedUserId)"
+          )
+      }
+
+    _ = self.blockUserProperty.signal
+      .combineLatest(with: project)
+      .takeWhen(blockUserEvent.values().ignoreValues())
```

Comment:
What does this line do? Just an interesting one to see `values().ignoreValues()`

Reply:
So we actually get an EmptyResponseEnvelope from the block user mutation or an error.  `blockUserEvent.values()` tells us that we got a response successfully and not an error. I'm using `.ignoreValues()` here since there isn't anything in the signal. 

Essentially, this block of code is saying, "React when `blockUserProperty` is triggered, combine its value with the late value in `project,` but only complete when the blocking mutation has returned a successful response."